src/main/java/com/epam/preprod/sirenko/Strategy.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+
+import java.math.BigDecimal;
+
+public interface Strategy {
+	PetGroup getPetGroup();
+	Season getSeason();
+	Size getSize();
+	String getStringName();
+	String getStringBrandName();
```

Comment:
Do we need 2 separate methods for string fields?

Reply:
couldn't find the way to do it with one method: in ManualProductCreationStrategy if I'll have one method I can't put two different requests like "enter name/brandName" in it